concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -64,82 +69,90 @@ def compute_task_def(
+    with freeze_time(timestamp):
+        return message.tasks.SubtaskResultsAccepted(
+            subtask_id = subtask_id,
+            payment_ts = payment_ts,
+        )
 
 
-def subtask_results_rejected(reason = None, report_computed_task = None):
-    return message.tasks.SubtaskResultsRejected(
-        reason                  = reason,
-        report_computed_task    = report_computed_task,
-    )
+def subtask_results_rejected(timestamp = None, reason = None, report_computed_task = None):
+    with freeze_time(timestamp):
+        return message.tasks.SubtaskResultsRejected(
+            reason                  = reason,
+            report_computed_task    = report_computed_task,
+        )
 
 
-def report_computed_task(subtask_id = None, task_to_compute = None):
-    return message.tasks.ReportComputedTask(
-        subtask_id      = subtask_id,
-        task_to_compute = task_to_compute
-    )
+def report_computed_task(timestamp = None, subtask_id = None, task_to_compute = None):
+    with freeze_time(timestamp):
+        return message.tasks.ReportComputedTask(
+            subtask_id      = subtask_id,
+            task_to_compute = task_to_compute
+        )
 
 
 def main():
     cluster_url     = parse_command_line(sys.argv)
     task_id         = random.randrange(1, 100000)
-    current_time    = int(datetime.datetime.now().timestamp())
+    current_time    = int(datetime.datetime.now().timestamp()) - 3600
 
     #  Test CASE 2A + 2D + 3 - Send ForceSubtaskResults with same task_id as stored by Concent before
     #  Step 1. Send ForceSubtaskResults first time
-    message_force_subtask_results = force_subtask_results(
-        ack_report_computed_task(
-            subtask_id = str(task_id) + '2A',
-            task_to_compute = task_to_compute(
-                compute_task_def = compute_task_def(
-                    task_id     = str(task_id) + '2A',
-                    subtask_id  = str(task_id) + '2A',
-                    deadline    = current_time + 5,
-                )
-            )
-        )
-    )
-    time.sleep(2)
     api_request(
         cluster_url,
         'send',
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
-        message_force_subtask_results,
+        force_subtask_results(
+            timestamp = get_strftime_from_timestamp(current_time),  # current_time
+            ack_report_computed_task = ack_report_computed_task(
+                timestamp = get_strftime_from_timestamp(current_time - 3601),  # current_time - 1:00:01
+                subtask_id = str(task_id) + '2A',
+                task_to_compute = task_to_compute(
+                    timestamp = get_strftime_from_timestamp(current_time - 3605),  # current_time - 1:00:05
+                    compute_task_def = compute_task_def(
+                        task_id     = str(task_id) + '2A',
+                        subtask_id  = str(task_id) + '2A',
```

Comment:
Please do that line 106 and reuse already prepared variable in all `task_id` and `subtask_id`.